test/expected/sql_query_results_optimized.out

```diff
@@ -190,6 +228,43 @@ LIMIT 2;
  Wed Dec 31 16:13:00 1969 | 809.5 | 10780 | 28.4500853206775
 (2 rows)
 
+--test on table with time partitioning function
```

Comment:
so this is not optimized (because there is an extra sort node as a child of limit. Please make a note in the comment.